src/systemctl/systemctl.c

```diff
@@ -1106,13 +1123,16 @@ static int systemctl_main(int argc, char *argv[]) {
         const Verb *verb = verbs_find_verb(argv[optind], verbs);
         if (verb && (verb->flags & VERB_ONLINE_ONLY) && arg_root)
```

Comment:
If the error message is being changed, presumably the condition should check for `arg_image` too?

Reply:
It's enough to just check for `arg_root` since we set `arg_root` when `arg_image` is set.